index.html

```diff
@@ -28,4 +28,5 @@
 	</div>
 </body>
 <script src="./app.js?v=0.2"></script>
+<script src="./geoip.js?v=0.2"></script>
```

Comment:
@lazyweirdo I am unable to inspect this file (github says its 54k of lines – impossible to review).
I think you've built browser bundle because `ipfs-geoip` does not provide one – was that the only reason?

Reply:
yes ... i packaged it, along with ipfs-http-client ... isn't it the correct way ?

Reply:
I see ... i'll give it a second chance ... trying to package only geoip without ipfs-http-client

Reply:
Git repo should not include any build outputs, only sources. 
Without sources, it is impossible to maintain it / change in the future. 

Let's wait for https://github.com/ipfs-shipyard/ipfs-geoip/pull/74 to land, it should make it possible to use `ipfs-geoip` the same way as you switched to prepackaged `ipfs-http-client` in a44ae00a9caaf49cfc6574bcd29c9f017c952226